lib/csharp/src/Thrift.sln

```diff
@@ -1,6 +1,8 @@
 ﻿
-Microsoft Visual Studio Solution File, Format Version 11.00
-# Visual Studio 2010
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 15
```

Comment:
Note that sometime in the future we're going to remove all provided static solution files and rely solely on project creation that comes with cmake, at which point you'll pick what you want at cmake generation time.

Reply:
is there any strong reason for this?